build.gradle

```diff
@@ -566,24 +509,6 @@ dependencyUpdates.resolutionStrategy = {
   }
 }
 
-configurations.configureEach {
-  resolutionStrategy.eachDependency { details ->
-    // Remedy for CVE-2020-8908, ensures launchdarkly uses correct version
-    if (details.requested.group == 'com.google.guava'
-      && details.requested.name == 'guava') {
-      details.useVersion "32.1.1-jre"
-    }
-  }
-
-  resolutionStrategy.eachDependency { details ->
-    // Remedy for CVE-2024-22233 - remove once spring-boot transitively uses version >= 6.1.14
-    if (details.requested.group == 'org.springframework'
-      && (details.requested.version == '6.1.12' || details.requested.version == '6.1.13')) {
-      details.useVersion "6.1.14"
-    }
-  }
-}
-
```

Comment:
please restore section and only update/remove strategies

Reply:
Restored, just fyi it is empty and looks a bit odd:
```
configurations.configureEach {
  resolutionStrategy.eachDependency { details ->
  }
}
```